samples/net/promiscuous_mode/src/main.c

```diff
@@ -77,7 +90,7 @@ static void print_info(struct net_pkt *pkt)
 	}
 
 	if (family == AF_UNSPEC) {
-		LOG_INF("Recv %p len %d (unknown address family)",
+		LOG_INF("Recv %p len %zd (unknown address family)",
```

Comment:
I wonder if this really works with minimal libc (or logging handles formatting itself?).